rpcserver.go

```diff
@@ -871,7 +865,7 @@ func (r *rpcServer) Start() error {
 			// through the following chain:
 			// req ---> CORS handler --> WS proxy --->
 			//   REST proxy --> gRPC endpoint
-			err := http.Serve(lis, restHandler)
+			err := http.Serve(lis, allowCORS(restHandler, r.cfg.RestCORS))
```

Comment:
nit: line too long. To make it a bit more explicit that another handler is added, you could write it:
```
corsHandler := allowCORS(restHandler, r.cfg.RestCORS)
err := http.Serve(lis, corsHandler)
```

Reply:
Fixed!